apps/api-server/src/routes/api/resource.js

```diff
@@ -81,6 +81,10 @@ router.all('*', function (req, res, next) {
     req.scope.push('includeTags');
   }
 
+  if (req.query.includeStatuses) {
```

Comment:
Kunnen we dit beter niet gewoon altijd aanwezig laten zijn, dat we die scope altijd meegeven als zijnde in de defaultscope? Een resource moet namelijk vgm altijd een status hebben dus het lijkt me niet nuttig om deze apart met een queryparam te includen

Reply:
Dat is al defaultScope. Die queryparam is zinloos; ik heb hem verwijderd.